### ISSUE TITLE
Have at least 1 high priority and 1 low priority worker per environment

### ISSUE THREAD

**Issue by mark-dce** [yalelibrary/YUL-DC#883]
**STORY**
We need a way to process near-realtime updates like changing a work's visibility even when long running jobs like PDF generation are running. 

**DETAILS**
Used named queues to do this

**ACCEPTANCE**
- [ ] We have at least one worker in every environment processing named queues for everything except PDFs
    - [x] Infra
    - [x] Test
    - [x] UAT
    - [x] Demo
    - [x] Staging
    - [x] Production
- [ ] We have at least one worker in every environment dedicated to a separate named queue for PDFs
    - [x] Infra
    - [x] Test
    - [x] UAT
    - [x] Demo
    - [x] Staging
    - [x] Production

**Comment by orangewolf** [comment]
I believe this is a matter of checking out the `named_workers` branch of camerata and then running `cam deploy-intensive-worker yul-dc-infra`

**Comment by orangewolf** [comment]
This requires the Jenkins update before we can create all the extra workers. Creating those workers makes this work confirmable.

**Comment by orangewolf** [comment]
Given the current state we think the next step is to deploy staging via Jenkins and see if the new workers are created successfully. There is a good expectation that they will be.

**Comment by bobrice** [comment]
Production now has one intensive worker.
